cleverhans/utils_tf.py

```diff
@@ -153,7 +156,7 @@ def model_train(sess, x, y, predictions, X_train, Y_train, save=False,
 
         for epoch in six.moves.xrange(args.nb_epochs):
             if verbose:
```

Comment:
If we are going to switch all of these prints to the log, it might make sense to deprecate the `verbose` argument and let users disable logging / not access the logs instead